Task5Test/src/test/java/com/company/TriangleTest.java

```diff
@@ -3,75 +3,107 @@
 import org.testng.Assert;
 import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
-
+import org.w3c.dom.Document;
+import org.w3c.dom.NamedNodeMap;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import java.io.File;
 import java.math.BigDecimal;
 
 public class TriangleTest {
 
+    private static final String VALID_TRIANGLES = "validTriangles";
+    private static final String INVALID_TRIANGLES = "invalidTriangles";
+    private static final String ISOSCELES_NODE = "isosceles";
+    private static final String EQUILATERAL_NODE = "equilateral";
+    private static final String PATH = "C:\\Users\\Kristina\\IdeaProjects\\Task5Test\\triangleData.xml";

```

Comment:
Absolute hardcoded path is bad as it doesn't allow to run unittests anywhere except your enviroment.